Microsoft.Toolkit.Uwp.UI.Controls/BladeView/BladeView.xaml

```diff
@@ -154,26 +341,28 @@
                                     TabIndex="0"
                                     HorizontalAlignment="Right"
                                     Background="{TemplateBinding CloseButtonBackground}"
-                                    BorderThickness="0"
+                                    BorderThickness="1"
```

Comment:
Why the change in border thickness? These were recently just changed to fit with new stlyes

Reply:
This enables a border to be put around the control in High Contrast themes. In Light/Dark this is a transparent border.

Reply:
But it increases the size of the border still. How about making that a theme resource and it's normally 0, but in high contrast it is 1

Reply:
Happy to do that. Will submit the change shortly.

Reply:
Thickness added to themes